packages/web/components/Modal/SettleDealModal/index.js

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useForm } from 'react-hook-form';
+import { ReviewButton as Button } from '../CurateTechnologyModal/styles';
+import { toast } from '../../Toast';
+import { Modal, InfosContainer, Summary } from './styles';
+import { CurrencyInputField, InputField } from '../../Form';
+import { formatCurrencyToInt, formatMoney } from '../../../utils/helper';
+
+const SettleDealModal = ({ closeModal }) => {
+	const [totalValue, setTotalValue] = useState(0);
+	const form = useForm();
+	const { watch } = form;
+	const formValues = watch();
+
+	const onSubmit = () => {
+		toast.success('Pedido fechado com sucesso');
+		closeModal();
+	};
+
+	useEffect(() => {
+		const { quantity, unityPrice } = formValues;
+
+		if (quantity && unityPrice) {
+			const intCurrency = formatCurrencyToInt(unityPrice);
+			const total = intCurrency * quantity;
+
+			setTotalValue(total);
+		}
+	}, [formValues]);
+
+	return (
+		<Modal onSubmit={form.handleSubmit(onSubmit)} noValidate>
+			<div>
+				<img
+					src="/checkout-rafiki.svg"
+					alt="Ilustração de uma moça registrando compras em um computador"
+				/>
+			</div>
+
+			<InfosContainer>
+				<h3>Deseja fechar este pedido?</h3>
+
+				<CurrencyInputField
+					form={form}
+					label="Valor unitário negociado"
+					name="unityPrice"
+					placeholder="R$ 0,00"
+					variant="gray"
+					validation={{ required: true }}
+				/>
+
```

Comment:
Just a tiny UI improvement:
![image](https://user-images.githubusercontent.com/10481028/98876360-e3100080-245c-11eb-8e10-2238834eaf13.png)
See how the required indicator is glued to the input field? Maybe it'd be better if we added an 8px~10px margin to the right of the label so they have a little room between each other:
![image](https://user-images.githubusercontent.com/10481028/98876465-1d799d80-245d-11eb-8fe7-1cab35da7e71.png)